core/designsystem/src/androidMain/kotlin/io/github/droidkaigi/confsched2022/designsystem/theme/Theme.kt

```diff
@@ -31,16 +34,21 @@ private val DarkColorPalette = darkColorScheme(
     outline = Color(KaigiColors.neutralVariantKeyColor60),
 )
 
+@SuppressLint("NewApi")
 @Composable
-fun KaigiTheme(
+public fun KaigiTheme(
     // Currently, we are not supporting light theme
 //    darkTheme: Boolean = isSystemInDarkTheme(),
+    isDynamicColorEnabled: Boolean = false,
     content: @Composable () -> Unit
 ) {
+    val colorScheme = if (isDynamicColorEnabled) {
+        dynamicDarkColorScheme(LocalContext.current)
```

Comment:
It seems difficult to apply dynamicLightColorScheme because Color.White is set in Text Composables 🤔 

<img src="https://user-images.githubusercontent.com/50792596/192679934-4993be40-ad77-4d93-897c-d0d668eebb7d.png" width="300" />